prowler/providers/aws/services/rds/rds_service.py

```diff
@@ -19,6 +19,7 @@ def __init__(self, provider):
         self.db_engines = {}
         self.db_cluster_parameters = {}
         self.db_cluster_snapshots = []
+        self.db_event_subscriptions = []
```

Comment:
Please, use a map here instead of a list.

Reply:
In this case I would leave a list

Reply:
If we have the ARN of the resource I'd use a map since then in the future we will get back to this to search for things and I think is simpler to do that now.

Reply:
You are right, but in this case, we are adding an empty resource to represent an empty region.